app.vue

```diff
@@ -1,6 +1,27 @@
+<script setup lang="ts">
+const message = apiRef({
+  route: '/api/auth/login',
+  method: 'post',
+  defaultValue: null
+});
+
+const { data } = await useFetch('/api/auth/login', {
+  method: 'post',
+  body: {
+    email: 'phu.nguyen2310@hcmut.edu.vn',
+    password: '123456789'
+  }
+});
+
+message.value = data.value;
```

Comment:
Can we merge these into one function call??

Reply:
> Can we merge these into one function call??

là seo

Reply:
like this 
```
const message = apiRef({
  route: '/api/auth/login',
  method: 'post',
  defaultValue: await useFetch('/api/auth/login', {
  method: 'post',
  body: {
    email: 'phu.nguyen2310@hcmut.edu.vn',
    password: '123456789'
  }
  }). value;
});
```